_faq/why-mixer.md

```diff
@@ -10,10 +10,28 @@ and the infrastructure backends used to perform access control checks and teleme
 layer enables operators to have rich insights and control over service behavior without requiring
 changes to service binaries.
 
-Mixer is designed as a stand-alone component, distinct from Envoy, in order to be able
-to manage scaling and reliability concerns independently . The proxy is focused on
-high-performance routing, while Mixer takes responsibility for policy management and telemetry collection.
-Mixer is invoked by Envoy, and in the future will also be invoked directly by Istio-based services.
+Mixer is designed as a stand-alone component, distinct from Envoy. This has numerous benefits:
+
+- *Scalability*.
+The work that Mixer and Envoy do is very different in nature, leading to different scalability
+requirements. Keeping the components separate enables independent component-appropriate scaling.
+
+- *Resource Usage*.
+Istio depends on being able to deploy many instances of its proxy, making it important to minimize the
+cost of each individual instance. Moving Mixer's complex logic into a distinct component makes it 
+possible for Envoy to remain svelte and agile.
+
+- *Reliability*.
+Mixer's operation is fairly complex, making it more susceptible to failures. Separating
+the components introduces distinct failure domains, enabling Envoy to continue
+operating even if Mixer fails.
+
+- *Isolation*. Mixer provides a level of insulation between Istio and the infrastructure backends. Each Envoy instance can be configured to have a 
+very narrow scope of interaction, limiting the impact of potential attacks.
+
+- *Extensibility*. It was imperative to design a simple extensibility model to allow Istio to interoperate
```

Comment:
Even more, this allows use of custom/alternate proxies without having to port policy enforcement and telemetry across systems.